sdk/compiler/daml-extension/src/extension.ts

```diff
@@ -285,8 +288,19 @@ function addIfInConfig(
 function getLanguageServerArgs(
   config: vscode.WorkspaceConfiguration,
   telemetryConsent: boolean | undefined,
+  sdkVersion: string | undefined,
 ): string[] {
-  const multiIDESupport = config.get("multiPackageIdeSupport");
+  let multiIDESupport = config.get("multiPackageIdeSupport");
+
+  // We'll say multi-ide is introduced in 2.9.0. The command existed
+  // in 2.8, but it was unfinished, so we shouldn't allow it to be used.
+  if (sdkVersion && sdkVersion != "0.0.0" && semver.lt(sdkVersion, "2.9.0")) {
```

Comment:
small suggestion, to avoid showing the warning when `multiPackageIdeSupport` is already off or undefined

```suggestion
  if (multiIDESupport && sdkVersion && sdkVersion != "0.0.0" && semver.lt(sdkVersion, "2.9.0")) {
```

Reply:
Oh! :facepalm: of course - good catch